src/components/Home/home.js

```diff
@@ -78,29 +85,44 @@ const Home = () => {
                   <Typography component="h1" variant="h6">
                     <strong>{currencyFormatter.format(a.balance)}</strong>
                   </Typography>
-                  <Button
-                    className='edit-button'
-                    type="button"
-                    fullWidth
-                    variant="contained"
-                    sx={{ mt: 3, mb: 2 }}
-                    onClick={() => handleEdit(a)}
-                  >
-                    <ModeEditIcon />
-                    Detalles
-                  </Button>
-                  <Button
-                    className='delete-button'
-                    type="button"
-                    fullWidth
-                    variant="contained"
-                    color="error"
-                    sx={{ mt: 3, mb: 2 }}
-                    onClick={() => { }}
-                  >
-                    <DeleteForeverIcon />
-                    Eliminar
-                  </Button>
+                  <Box>
+                    <Button
+                      className='edit-button'
+                      type="button"
+                      fullWidth
+                      variant="contained"
+                      sx={{ mt: 3, mb: 2 }}
+                      onClick={() => handleEdit(a)}
+                    >
+                      <ModeEditIcon />
+                      Detalles
+                    </Button>
+                    <Button
+                      className='delete-button'
+                      type="button"
+                      fullWidth
+                      variant="contained"
+                      color="error"
+                      sx={{ mt: 3, mb: 2 }}
+                      onClick={() => { }}
+                    >
+                      <DeleteForeverIcon />
+                      Eliminar
+                    </Button>
+                  </Box>
+                  <Box>
+                    <Button
+                      className='transactions-button'
+                      type="button"
+                      variant="contained"
+                      color="success"
+                      sx={{ mt: 3, mb: 2 }}
+                      onClick={() => goToTransactions(`/accounts/${a.id}/transactions`)}
```

Comment:
Non blocking: it's always a good idea to use more descriptive names for the variables, instead of `a` you could use `account`